socs/plugin.py

```diff
@@ -41,6 +41,7 @@
     'SmurfFileEmulator': {'module': 'socs.agents.smurf_file_emulator.agent', 'entry_point': 'main'},
     'SmurfStreamSimulator': {'module': 'socs.agents.smurf_stream_simulator.agent', 'entry_point': 'main'},
     'SmurfTimingCardAgent': {'module': 'socs.agents.smurf_timing_card.agent', 'entry_point': 'main'},
+    'StmThermometerAgent': {'module': 'socs.agents.stm_thermo.agent', 'entry_point': 'main'},
```

Comment:
Needs updating for the new Agent class name and module path.

```suggestion
    'StimThermometerAgent': {'module': 'socs.agents.stimulator_thermometer.agent', 'entry_point': 'main'},
```